packages/analytics-js-integrations/src/integrations/DCMFloodlight/browser.js

```diff
@@ -79,52 +80,39 @@ class DCMFloodlight {
    * Ref - https://developers.google.com/authorized-buyers/rtb/cookie-guide
    */
   loadCookieMatching() {
-    logger.debug('===In loadCookieMatching DCMFloodlight===');
     if (this.doubleclickId && this.googleNetworkId) {
       const image = document.createElement('img');
-      image.src = `https://cm.g.doubleclick.net/pixel?google_nid=${
-        this.googleNetworkId
-      }&google_hm=${btoa(this.analytics.getAnonymousId())}`;
+      image.src = `https://cm.g.doubleclick.net/pixel?google_nid=${this.googleNetworkId
+        }&google_hm=${btoa(this.analytics.getAnonymousId())}`;
       document.getElementsByTagName('head')[0].appendChild(image);
     }
   }
 
   isLoaded() {
-    logger.debug('===In isLoaded DCMFloodlight===');
     if (this.tagFormat === GTAG) {
       return window.dataLayer.push !== Array.prototype.push;
     }
     return true;
   }
 
   isReady() {
-    logger.debug('===In isReady DCMFloodlight===');
-    if (this.tagFormat === GTAG) {
-      return window.dataLayer.push !== Array.prototype.push;
-    }
-    return true;
-  }
-
-  identify() {
```

Comment:
if identify is not supported, what error would they get if this handling is removed ?

Reply:
they will never get any error. if we remove identify code then core sdk will not even call destination.identify()

Reply:
Is it handled in both v1.1 & v3 ?

Reply:
done reverted this change